README.md

```diff
@@ -40,6 +40,19 @@ $ cd cardano-node
 $ cabal build
 ```
 
+## Windows Executable
+
+### Download
+You can download [here](https://hydra.iohk.io/job/Cardano/cardano-node/cardano-node-win64/latest-finished).
+
+### Instructions
+
+The download includes cardano-node.exe and a .dll. To run the node with cardano-node run you need to reference a few files and directories as arguments. These can be copied from the cardano-node repo into the executables directory. The command to run the node on mainnet looks like this:
```

Comment:
Maybe add a link to config directory where those files can be found: https://github.com/input-output-hk/cardano-node/tree/master/configuration or just mention that they are all in `configuration` directory.